src/qtgui/plotter.cpp

```diff
@@ -850,7 +850,7 @@ void CPlotter::resizeEvent(QResizeEvent* )
         m_2DPixmap = QPixmap(m_Size.width(), fft_plot_height);
         m_2DPixmap.fill(Qt::black);
 
-        int height = (100 - m_Percent2DScreen) * m_Size.height() / 100;
```

Comment:
Here the integer division rounds the result down, typically resulting in a blank pixel at the bottom of the waterfall.